src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Persistence/ResourceHandlerTests.cs

```diff
@@ -121,7 +121,7 @@ public async Task GivenAFhirMediator_WhenSavingAResource_ThenLastUpdatedShouldBe
 
                 resource = (await _mediator.UpsertResourceAsync(resource)).Resource;
 
-                Assert.Equal(instant, resource.LastUpdated);
+                Assert.Equal(new DateTimeOffset(DateTimeOffset.Now.Date.AddMilliseconds(6), TimeSpan.Zero), resource.LastUpdated);
```

Comment:
This could potentially fail around midnight?

Reply:
Yes, I should be snapshotting the `.Now` value. Good call!